Mapping/Reader/Annotation.php

```diff
@@ -66,6 +111,8 @@ public function read($class)
             return [];
         }
 
+        /** config properties: columns (string[]), commentProperty (string),  */
```

Comment:
```suggestion

        /** 
         * @var string[] $config['columns'] Some 
         * @var string[] $config['commentsProperty'] Some 
         */
```
Такой вариант вполне пхпшторм и читается проще, и парсится IDE. В остальном от меня ок.